assets/js/settings/blocks/constants.ts

```diff
@@ -39,3 +39,19 @@ export const CART_URL = STORE_PAGES.cart.permalink;
 export const LOGIN_URL = STORE_PAGES.myaccount.permalink
 	? STORE_PAGES.myaccount.permalink
 	: getSetting( 'wpLoginUrl', '/wp-login.php' );
+export const SHIPPING_COUNTRIES = getSetting(
+	'shippingCountries',
+	{}
+) as Record< string, string >;
+export const ALLOWED_COUNTRIES = getSetting( 'AllowedCountries', {} ) as Record<
```

Comment:
`AllowedCountries` shouldn't be capitalized:
```suggestion
export const ALLOWED_COUNTRIES = getSetting( 'allowedCountries', {} ) as Record<
```

Reply:
Thanks Albert, good spot 🕵🏼